cfpb/cfgov-refresh
# Change validation messages for D&R conference module and JS

[Per request](https://github.cfpb.gov/CFGOV/data-research/issues/320#issuecomment-90362) from @schaferjh and @marteki, this slightly changes the form validation messages a user receives.

## Changes

- Form validation messages for the D&R conference module and the JS

## Testing

- Run `gulp test` 
- Add a conference registration module to a browse page, and submit successfully by adding a valid GD code, and then change the code to something invalid and submit with that to see the other changed message.

## Review

- @cfpb/cfgov-frontends to check the form validation message change
- @cfpb/cfgov-backends to check the rest

## Notes

- @Scotchester should we also change the Feedback module? Possibly [here.](https://github.com/cfpb/cfgov-refresh/blob/change-form-validation-messages/cfgov/v1/handlers/blocks/feedback.py#L48)

## Checklist
